bindings/nodejs/lib/types/utils/bridge/utils.ts

```diff
@@ -224,7 +224,7 @@ export interface __VerifyTransactionSemantic__ {
         inputs: InputSigningData[];
         protocolParameters: ProtocolParameters;
         unlocks?: Unlock[];
-        manaRewards?: { [outputId: OutputId]: NumericString };
+        manaRewards?: { [outputId: HexEncodedString]: NumericString };
```

Comment:
we definitely can't use OutputId here? I know it can't be used in the dict but is there another way?

Reply:
No idea what other way there could be

Reply:
there is a `Map` type in js, could probably use that instead: 
```js
manaRewards?: Map<OutputId, NumericString>;
```

Reply:
I think there were some issues with the Map type, for example the serialization by default ends up in an empty object
And for things with a BTreeMap we also undid it here https://github.com/iotaledger/iota-sdk/pull/501

Reply:
:(